app/components/form/history.hbs

```diff
@@ -0,0 +1,38 @@
+{{#if this.loading}}
+  Loading...
+{{else}}
+  <AuTable @size={{this.size}}>
+    <:header>
+      <tr>
+        <th>Tijdstip</th>
+        <th>Gebruiker</th>
+        <th>Aanpassing</th>
+        <th></th>
+      </tr>
+    </:header>
+    <:body>
+      {{#each this.history as |historyItem index|}}
+        <Form::HistoryRow
+          @historyItem={{historyItem}}
+          @onRestore={{perform this.restoreHistoryItem}}
+          @isCurrentVersion={{eq index 0}}
```

Comment:
This does not work if you have multiple pages
![Screenshot 2024-03-19 at 16 30 41](https://github.com/lblod/frontend-lokaal-mandatenbeheer/assets/43848896/f1bd7a9a-6dd5-41fd-8b34-681d23ab5120)